os/Makefile.unix

```diff
@@ -686,3 +686,15 @@ romfs:
 ifeq ($(CONFIG_FS_ROMFS),y)
 	$(Q) ../tools/fs/mkromfsimg.sh
 endif
+
+# compression_tool
+#
+# The compression_tool target will make a compression tool by calling mkcompresstool.sh.
+# This compression tool (mkcompressimg) will be called to compress binaries.
+# Hence, this tool needs to be created before we build those binaries.
+
+compression_tool:
```

Comment:
How about using same naming of target?
```
tools/mkdeps
```
Because I can understand easily with this target name. It's located at tools folder and makes "making deps" tool.

But `compression_tool` does not show that it's calling compression tool or making tool or something else.

Reply:
Ok..  will make this change